### PR TITLE
Enhancement: Remove Easy py2-isms

### PR DESCRIPTION
**Pull request by brolewis**
Make changes that remove (easy) Python 2-isms to bring us into **THE MODERN AGE**

### CODE REVIEW

File: sphinx_gherkindoc/cli.py [hunk 1]
```diff
@@ -163,24 +163,22 @@ def main() -> None:
...
-        )
+        parser.exit(message=f"Sphinx (sphinx-gherkindoc) {sphinx.__display_version__}")
 
     set_dry_run(args.dry_run)
     set_verbose(args.verbose)
 
     gherkin_path = pathlib.Path(args.gherkin_path)
     if not gherkin_path.is_dir():
-        parser.error("{} is not a directory.".format(gherkin_path))
+        parser.error(f"{args.gherkin_path} is not a directory.")
```
Comment (dgou) [inline sphinx_gherkindoc/cli.py]: Why the `args.` prefix?  One of the "great joys" I've had with debugging relative path programs is in getting errors like this and not having any clue what the actual path being complained about is...  Probably even happened while working on this very package when I was actively working on it...

File: sphinx_gherkindoc/utils.py [hunk 2]
```diff
@@ -39,9 +39,9 @@ def set_verbose(value: bool) -> None:
 
 
 # Build up dictionary of characters that need escaping
-_escape_mappings = {ord(x): u"\\{}".format(x) for x in ("*", '"', "#", ":", "<", ">")}
+_escape_mappings = {ord(x): f"\\{x}" for x in ("*", '"', "#", ":", "<", ">")}
 _advanced_escape_mappings = _escape_mappings.copy()
-_advanced_escape_mappings[ord("\\")] = u"\\\\\\"
+_advanced_escape_mappings[ord("\\")] = "\\\\\\"
```
Comment (dgou) [inline sphinx_gherkindoc/utils.py]: I saw what `u` did there!

File: sphinx_gherkindoc/utils.py [hunk 3]
```diff
@@ -104,7 +104,8 @@ def add_output(self, line: str, line_breaks: int = 1, indent_by: int = 0) -> Non
             line_breaks: The number of line breaks to include
             indenty_by: The number of spaces to indent the line.
         """
-        self._output.append(u"{}{}{}".format(" " * indent_by, line, "\n" * line_breaks))
+        line_breaks_str = "\n" * line_breaks
+        self._output.append(f"{' ' * indent_by}{line}{line_breaks_str}")
```
Comment (dgou) [inline sphinx_gherkindoc/utils.py]: Nice, the intermediate variable makes for greater clarity even though it adds a line.

File: sphinx_gherkindoc/cli.py (hunk 1 above)
Comment (brolewis) [reply sphinx_gherkindoc/cli.py]: So the intention here (as with https://github.com/jolly-good-toolbelt/sphinx_gherkindoc/pull/8/files#diff-27df03292cffa651fb903656cef3cde4R181) was to make the error message return to the user what they provided to the value so there isn't confusion over "what is that path? That's not what I typed"

Comment (dgou) [reply sphinx_gherkindoc/cli.py]: Yeah, i get that too, but for a "just convert to f-strings" it feels a little sleezy to sneak that in.

**Pull request merged**